scripts/generate_sitemap/index.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'path';
+import { concat } from 'lodash';
+import { simpleSitemapAndIndex } from 'sitemap';
+import {
+  getLearnPageItems,
+  getTopLevelPageItems,
+  getLocationPageItems,
+} from './utils';
+
+const destinationDir = path.join(__dirname, '../../build');
+
+function main() {
+  console.log('Generating the sitemap');
+  const topLevelPageItems = getTopLevelPageItems();
+  const locationPageItems = getLocationPageItems();
+  const learnPageItems = getLearnPageItems();
+
+  const urlList = concat(topLevelPageItems, locationPageItems, learnPageItems);
+  console.log(`Total URLs: ${urlList.length}`);
+
+  simpleSitemapAndIndex({
```

Comment:
This returns a promise but we're ignoring it, which probably means:
1. We'll log 'Done' before the sitemap is generated.
2. If there are any errors, they won't fail the script.

I think we need to add:

```
.catch(err => {
  console.error(err);
  process.exit(-1);
}).then(() => { 
  console.log('Done.');
});
```

or else rework it to use async/await.  Note that if we use async / await we'll still need a top-level `catch` handler that calls `process.exit`, similar to https://github.com/covid-projections/covid-projections/blob/267ff9ea4155fb17f6eb78012c5b7f95c12283bd/scripts/generate_index_pages.ts#L263